spec/support/fetchable_behavior.rb

```diff
@@ -136,3 +173,45 @@
     end
   end
 end
+
+shared_examples_for 'a record with an indexable url' do
+  describe 'validations' do
+    it { is_expected.to allow_value("http://some.site.gov/url").for(:url) }
+    it { is_expected.to allow_value("http://some.site.mil/").for(:url) }
+    it { is_expected.to allow_value("https://some.govsite.info/url").for(:url) }
+
+    context 'when the url extension is blacklisted' do
+      let(:movie_url) { "http://www.nps.gov/some.mov" }
+      let(:record) { described_class.new(valid_attributes.merge(url: movie_url)) }
+
+      it "is not valid" do
+        expect(record).not_to be_valid
+        expect(record.errors.full_messages.first).to match(/extension is not one we index/i)
+      end
+    end
+  end
+end
+
+shared_examples_for 'a record with a searchgov_domain' do
```

Comment:
Please rename this to:
`shared_examples_for 'a record that belongs to a searchgov_domain'`

`a record with` seems to indicate a `has_one` relationship.